docs/wiki/Solution creation.md

```diff
@@ -0,0 +1,302 @@
+This section shows you how you can orchestrate a deployment using multiple resource modules.
+
+> **Note:** For the sake of the below examples we assume you leverage Bicep as your primary DSL.
+
+- [Template-orchestration](#template-orchestration)
+
+# Template-orchestration
+
+The _template-orchestrated_ approach means using a _main_ or so-called _master template_ for deploying resources in Azure. This template will only contain nested deployments, where the modules – instead of embedding their content into the _master template_ – will be referenced by the _master template_.
+
+With this approach, modules need to be stored in an available location, where the Azure Resource Manager (ARM) can access them. This can be achieved by storing the modules templates in an accessible location location like _local_, _template specs_ or the _bicep registry_.
```

Comment:
```suggestion
With this approach, modules need to be stored in an available location, where the Azure Resource Manager (ARM) can access them. This can be achieved by storing the modules templates in an accessible location like _local_, _template specs_ or the _bicep registry_.
```